cypress/e2e/item/duplicate/gridDuplicateItem.cy.ts

```diff
@@ -0,0 +1,50 @@
+import { HOME_PATH, buildItemPath } from '../../../../src/config/paths';
+import {
+  ITEM_MENU_DUPLICATE_BUTTON_CLASS,
+  buildItemCard,
+  buildItemMenu,
+  buildItemMenuButtonId,
+} from '../../../../src/config/selectors';
+import { ITEM_LAYOUT_MODES } from '../../../../src/enums';
+import { SAMPLE_ITEMS } from '../../../fixtures/items';
+
+const duplicateItem = ({ id }: { id: string }) => {
+  const menuSelector = `#${buildItemMenuButtonId(id)}`;
+  cy.get(menuSelector).click();
+  cy.get(`#${buildItemMenu(id)} .${ITEM_MENU_DUPLICATE_BUTTON_CLASS}`).click();
+};
+
+describe('Duplicate Item in Grid', () => {
+  it('Duplicate item on Home', () => {
+    cy.setUpApi(SAMPLE_ITEMS);
+    cy.visit(HOME_PATH);
+    cy.switchMode(ITEM_LAYOUT_MODES.GRID);
+
+    // Duplicate
+    const { id: duplicateItemId } = SAMPLE_ITEMS.items[0];
+    duplicateItem({ id: duplicateItemId });
+
+    cy.wait('@copyItems').then(({ request: { url } }) => {
```

Comment:
Let's check the body as well, that it should contain the same id.

Reply:
In case of duplicating item on home, body will be an empty object, SO I just added test when duplicating item in an item

Reply:
It's still important to check that the body is empty, otherwise the item gets copy at another place.